shoop/core/management/commands/shoop_extractmessages.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+"""
```

Comment:
Then embetter the external script to do what it should, imho.

I don't feel a management command is the right fit for this job; management commands are supposed to be run in the context of a finished project, and i18n extraction for the Shoop codebase shouldn't require a project (be it `shoop_workbench` or anything else).